package.json

```diff
@@ -52,31 +48,31 @@
     "husky": "^3.1.0",
     "mocha": "^9.2.0",
     "nodemon": "^2.0.15",
-    "prettier": "^2.5.1",
+    "prettier": "^2.8.0",
     "testcontainers": "^1.1.19",
-    "ts-node": "^8.3.0",
-    "tslint": "^5.18.0",
-    "typescript": "^3.5.3"
+    "ts-node": "^10.0.0",
+    "typescript": "^5.0.0"
   },
   "dependencies": {
     "ajv": "^6.12.6",
-    "axios": "^0.26.0",
+    "axios": "^1.0.0",
     "body-parser": "^1.19.2",
-    "dotenv": "^8.0.0",
+    "dotenv": "^16.0.0",
     "errorhandler": "^1.5.1",
-    "express": "^4.17.3",
+    "express": "^4.18.2",
     "jsonwebtoken": "^8.5.1",
     "lodash": "^4.17.21",
     "memoizee": "^0.4.15",
-    "mongoose": "^5.13.14",
+    "mongoose": "^7.0.0",
     "ms": "^2.1.3",
     "node-vault": "^0.9.22",
     "swagger-ui-express": "^4.3.0",
     "winston": "^3.6.0"
   },
   "prettier": {
-    "printWidth": 100,
+    "printWidth": 120,
     "trailingComma": "all",
-    "singleQuote": true
+    "singleQuote": true,
+    "arrowParens": "always"
```

Comment:
should this include the use tabs param?

Reply:
I'll do a separate PR and fix all the prettier stuff. thanks.